swarm/storage/mru/handler.go

```diff
@@ -0,0 +1,509 @@
+// Copyright 2018 The go-ethereum Authors
```

Comment:
how can i see the diff here? 
This is not what we discussed or?

Reply:
you skipped steps!